tests/system/test_helpers.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 The PyBigQuery Authors
+#
+# Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+import os
+import json
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def module_under_test():
+    from pybigquery import _helpers
+
```

Comment:
What's he purpose of this?

Why not just refer to `_helpers`? Do we think the name is likely to change?